engine/installation/linux/centos.md

```diff
@@ -256,13 +279,27 @@ a new file each time you want to upgrade Docker.
     $ sudo yum install /path/to/package.rpm
     ```
 
-3.  Start Docker.
+3.  Edit `/etc/docker/daemon.json`. If it does not yet exist, create it. Assuming
+    that the file was empty, add the following contents.
+
+    ```json
+    {
+      "storage-driver": "devicemapper"
+    }
+    ```
+
+4.  For production systems, you must use `direct-lvm` mode, which requires you
```

Comment:
I'm worried that we re-introduce device mapper in the standard installation steps here. We moved device mapper below overlay and overlay2 exactly to prevent this situation; i.e. running with overlay/overlay2 is highly recommended over using device mapper on loopbacks.

People _will_ skip the "direct-lvm" steps, and run on loopbacks; get really bad performance, and as a result, tons of issues reported.

If device mapper is the only configuration we support for EE, and we need instructions for setting up Docker EE in a configuration that qualifies to get technical support, I think we should have a separate installation guide for Docker EE that explains all the steps to do this, instead of trying to combine EE and CE installation.